src/SearchResult.test.tsx

```diff
@@ -51,44 +45,24 @@ describe("SearchResult", () => {
 
   describe("with URL", () => {
     it("should render an anchor tag that opens in the same window by default", () => {
-      render(
-        <SearchResult searchResult={MOCK_SEARCH_RESULT} isSelected={false} />
-      );
-      const anchor = screen
-        .getByText(MOCK_SEARCH_RESULT.snippet.text)
-        .closest("a");
+      render(<SearchResult searchResult={MOCK_SEARCH_RESULT} isSelected={false} />);
+      const anchor = screen.getByText(MOCK_SEARCH_RESULT.snippet.text).closest("a");
 
       expect(anchor).toHaveAttribute("href", MOCK_SEARCH_RESULT.url);
       expect(anchor).toHaveAttribute("target", "_self");
     });
 
     it("should render an anchor tag that opens in a new tab if configured", () => {
-      render(
-        <SearchResult
-          searchResult={MOCK_SEARCH_RESULT}
-          isSelected={false}
-          shouldOpenInNewWindow={true}
-        />
-      );
-      const anchor = screen
-        .getByText(MOCK_SEARCH_RESULT.snippet.text)
-        .closest("a");
+      render(<SearchResult searchResult={MOCK_SEARCH_RESULT} isSelected={false} shouldOpenInNewTab={true} />);
+      const anchor = screen.getByTestId("vectara-react-search-result-link");
```

Comment:
Nit: I think it's safe to use a simpler prefix for this to avoid collisions. I've also been using camel-casing for consistency with how we've been naming CSS classes and JS vars. For example this could be `vrsSearchResultLink` or `vrsResultLink`, which are a bit terser.

Reply:
https://github.com/vectara/react-search/pull/22/commits/8711b05a1fd7913806accfe756898908baeaf09f